yarl/_path.py

```diff
@@ -39,3 +40,24 @@ def normalize_path(path: str) -> str:
 
     segments = path.split("/")
     return prefix + "/".join(normalize_path_segments(segments))
+
+
+def relative_path(path: str, start: str) -> str:
+    """A wrapper over os.path.relpath()"""
```

Comment:
Please use a verb to describe what the function does instead of a reference to some unidentified external location. This will help reduce the mental overhead for the readers.